app/screens/TransactionAuthorization.tsx

```diff
@@ -309,7 +305,6 @@ async function execWithAutoRetries (promptPromise: () => Promise<any>, onAutoRet
     return await promptPromise()
   } catch (e) {
     Logging.error(e)
-    retries = await PasscodeAttemptCounter.get() ?? 0
```

Comment:
Remove persistent storage access from here.

1. SecureStore is a relatively expensive/heavy resource to be accessed (quite slow), submit via state is much more efficient
2. trying to minimize the risk if any uncaught error here, potentially lead to unhandled statement management